frontend/components/Button/Button.css

```diff
@@ -22,12 +22,17 @@
   font-size: 15px;
   text-decoration: none;
   border-radius: 2px;
-  transition: background-color 0.1s, padding 0.3s, background-position 0.3s;
+  transition: background-color 0.1s,
+              box-shadow 0.1s,
+              padding 0.3s,
+              background-position 0.3s;
   position: relative;
 }
 
 .button:hover {
-  box-shadow: 0 -3px 0 0 rgba(0,0,0,0.05) inset, 0 0 5px 0px rgba(0,0,0,0.15);
+  box-shadow: 10000px 0 0 rgba(0,0,0,0.1) inset,
```

Comment:
@eduardoboucas do you agree with this approach rather than the individual static shade variables?

Reply:
Sure, looks great.